### DISCUSSION TITLE
SMTP Error: Could not connect to SMTP host. Connection failed

### DISCUSSION THREAD

**Discussion by Paolo027** [PHPMailer/PHPMailer#2892] [Q&A]
Hi Everyone,

I would like to ask what could be causing the errors that I am getting. Kindly see the error logs below:

2023-04-09 13:25:36 SERVER -> CLIENT: 220 2.0.0 Ready to start TLS
SMTP Error: Could not connect to SMTP host. Connection failed. stream_socket_enable_crypto(): SSL: An existing connection was forcibly closed by the remote host
2023-04-09 13:26:18 CLIENT -> SERVER: QUIT
2023-04-09 13:26:18 SERVER -> CLIENT:
2023-04-09 13:26:18 SMTP ERROR: QUIT command failed:
SMTP Error: Could not connect to SMTP host. Connection failed. stream_socket_enable_crypto(): SSL: An existing connection was forcibly closed by the remote host
Message could not be sent. Mailer Error: SMTP Error: Could not connect to SMTP host. Connection failed. stream_socket_enable_crypto(): SSL: An existing connection was forcibly closed by the remote hostSMTP server error: QUIT command failed

My code seems to be okay because I just used the sample code from "Simple Example" section. Thank you !

**Comment by Synchro** [comment]
What happened when you tried the tests in the troubleshooting guide, for example using openssl?

**Comment by omocinteractive** [comment]
Here the same problem (office365).
